x-pack/plugin/shutdown/src/main/java/org/elasticsearch/xpack/shutdown/ShutdownPlugin.java

```diff
@@ -90,4 +92,9 @@ public List<RestHandler> getRestHandlers(
     ) {
         return Arrays.asList(new RestPutShutdownNodeAction(), new RestDeleteShutdownNodeAction(), new RestGetShutdownStatusAction());
     }
+
+    @Override
+    public void handleTerminationSignal(NodeClient client, NodeEnvironment nodeEnvironment) {
+        // this space intentionally left blank
```

Comment:
is this comment meant to be a permanent blank, or a placeholder for followup work? A little more explicit comment  here would help future readers.